eng/scripts/vs.17.intpreview.json

```diff
@@ -13,10 +13,13 @@
         "Microsoft.VisualStudio.Component.VC.ATL.ARM64",
         "Microsoft.VisualStudio.Component.VC.Tools.ARM64",
         "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.ARM64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.x86.x64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL.ARM64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL",      
```

Comment:
```suggestion
        "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL",
```

Reply:
Good eye, may as well clean it up both files while we're here.